src/fixate/config/__init__.py

```diff
@@ -29,11 +29,12 @@
 # Begin default "plugins"
 # Use plg_ prefix with dictionary of values to indicate to fixate to install this at startup
 # Default settings for csv reporting. Can be configured via yaml either removing or overriding with plg_csv
+tpl_csv_path = ["{start_date_time}-{index}.csv"]
+tpl_time_stamp = "{0:%Y}{0:%m}{0:%d}-{0:%H}{0:%M}{0:%S}"
+
```

Comment:
Doesn't need fixing now, but I think we need a log file name template and a log path. I'll raise an issue and we can tidy up later.

Reply:
Or maybe it's just need to be documented. See #21 for more thoughts